splinepy/microstructure/tiles/snappy.py

```diff
@@ -61,7 +65,9 @@ def _closing_tile(
         if closure is None:
             raise ValueError("No closing direction given")
 
-        self.check_params(parameters)
+        # TODO: parameters are not implemented, therefore do not check params
+        if parameters is not None:
```

Comment:
The parameters for this tile are not implemented. Right now the code does not reach these lines when `parameters` are defined, since `def _closing_tile` is run after the creation of the other tiles. It would raise a not implemented error in the `def create_tile()` before that. So technically, this line is wrong, but it never reaches that line.

Thanks for catching that. Should I change that?